apps/io-services-cms-webapp/openapi.yaml

```diff
@@ -132,12 +132,6 @@ paths:
       summary: Retrieve all services
       description: Retrieve all service topics
       operationId: getServiceTopics
-      parameters:
-        - $ref: '#/components/parameters/UserEmail'
-        - $ref: '#/components/parameters/UserGroups'
-        - $ref: '#/components/parameters/UserID'
-        - $ref: '#/components/parameters/SubscriptionID'
-        - $ref: '#/components/parameters/XForwardedFor'
```

Comment:
**chore:** from the _Services CMS_ point of view, by removing _XForwardedFor_ we "lost" the origin caller IP....Azure will automatically add to this header only the last caller IP (i.e. the _Backoffice_ IP)